src/sentry/static/sentry/app/views/releases/detail/releaseHeader.tsx

```diff
@@ -114,11 +122,34 @@ const ReleaseHeader = ({
               <Count value={sessionsCrashed} />
             </ReleaseStat>
           )}
+          <Feature features={['release-performance-views']}>
+            <ReleaseStat
+              label={t('Apdex Score')}
+              help={getTermHelp(organization, 'apdex')}
+            >
+              <DiscoverQuery
+                eventView={releaseEventView}
+                location={location}
+                orgSlug={organization.slug}
+              >
+                {({isLoading, error, tableData}) => {
+                  if (isLoading || error || !tableData || tableData.data.length === 0) {
+                    return '\u2014';
+                  }
+                  return (
+                    <Count
+                      value={tableData.data[0][`apdex_${organization.apdexThreshold}`]}
```

Comment:
```suggestion
                      value={tableData.data[0][getAggregateAlias(`apdex(${organization.apdexThreshold})`)]}
```
Thoughts on using `getAggregateAlias` here rather than hard coding the alias name?